ckanext/scheming/templates/scheming/snippets/display_field.html

```diff
@@ -5,9 +5,14 @@
 
   {% for field_data in fields %}
     <div class="panel panel-default">
-      <header class="panel-heading">
-        {{ h.scheming_language_text(field.label) }} {{ loop.index }}
-      </header>
+      {% if loop.index == fields|length %}
+        <header class="panel-heading">
+          {{ h.scheming_language_text(field.label) }} {{ loop.index }}
+        </header>
+      {%- else -%}
+        <details>
+          <summary class="panel-heading">{{ h.scheming_language_text(field.label) }} {{ loop.index }}</summary>
+      {%- endif -%}
```

Comment:
this change should go in ckanext-canada, it's a preference for how we display our status field not a sensible default for all composite field users

Reply:
ckanext-canada doesn't seem to have a display_field.html snippet. Would adding one override the current one in scheming?

Reply:
yes we load our plugin first so it would override the scheming one. You can do the same with the confirmation dialog if you want to override the bootstrap one with a wet-boew one. `{% ckan_extends %}` and `{% block %}` can be used to override specific sections instead of the whole snippet/template